exchanges_tests/test_bitget.py

```diff
@@ -0,0 +1,67 @@
+#  This file is part of OctoBot (https://github.com/Drakkar-Software/OctoBot)
+#  Copyright (c) 2022 Drakkar-Software, All rights reserved.
+#
+#  OctoBot is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  OctoBot is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public
+#  License along with OctoBot. If not, see <https://www.gnu.org/licenses/>.
+import decimal
+
+import pytest
+
+from exchanges_tests import abstract_authenticated_exchange_tester
+
+# All test coroutines will be treated as marked.
+pytestmark = pytest.mark.asyncio
+
+
+class TestBitgetAuthenticatedExchange(
+    abstract_authenticated_exchange_tester.AbstractAuthenticatedExchangeTester
+):
+    # enter exchange name as a class variable here
+    EXCHANGE_NAME = "bitget"
+    ORDER_CURRENCY = "BTC"
+    SETTLEMENT_CURRENCY = "USDT"
+    SYMBOL = f"{ORDER_CURRENCY}/{SETTLEMENT_CURRENCY}"
+    ORDER_SIZE = 40  # % of portfolio to include in test orders
+    CONVERTS_ORDER_SIZE_BEFORE_PUSHING_TO_EXCHANGES = True
+    IGNORE_ORDER_FEE = True
```

Comment:
Why should we ignore fee here?

Reply:
fees are not always provided by the exchange